nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -37,4 +39,13 @@
      * @return the command status corresponding to the command identifier.
      */
     BulkStatus getStatus(String bulkId);
+
+    /**
+     * Waits for completion of given bulk.
```

Comment:
if a command is an action applied to a document set, you should wait for a command using a commandId, for me it is not clear that a bulkId is BulkCommand#getId